containers/codespaces-linux/.devcontainer/setup-user.sh

```diff
@@ -35,10 +35,7 @@ sudo -u ${USERNAME} git config --global core.editor "/home/${USERNAME}/.local/bi
 rm -f /tmp/scripts/git-ed.sh
 
 # Add one time notice
-tee -a /etc/bash.bashrc /etc/zsh/zshrc << EOF
-if [ -t 1 ] && [ ! -f \$HOME/.config/vscode-dev-containers/first-run-notice ]; then
-  echo -e "Welcome to Codespaces! Note that the default Codespaces image is now Ubuntu 20.04-based!\nIf you need to use the old image (or a custom one) see https://aka.ms/ghcs-default-focal."
-  mkdir -p \$HOME/.config/vscode-dev-containers
-  touch \$HOME/.config/vscode-dev-containers/first-run-notice
-fi
+cat << 'EOF' > "/usr/local/etc/vscode-dev-containers/first-run-notice.txt"
```

Comment:
What's your thinking around how long we should offer this update help text? I was thinking about a month, but maybe we want to go longer? I'm just not sure how relevant this will be for new users who are onboarding to Codespaces.

Reply:
Given most users are going to be returning right now, I left it. Adoption takes time since existing Codespaces will still be on the old image, so I was more thinking 2-3 mos for a big update like this.

I did, however, also add text into codespaces-linux-stretch that calls it "depricated" and points the other direction now.  So I was thinking we could leave it a month with both and then remove this one.  That's one of the reasons I moved it into common code for future proofing.

@2percentsilk any thoughts from your perspective on when we should consider dropping the notice in the Focal image?

Reply:
Let's merge this "as-is" for now to decouple the message conversation.  We can easily create a new PR just for that part.